protos/channel_responses.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+/* Following chromium style (e.g. sync_pb) is the usage for the sync protobuf
+*/
+package publishers_pb;
+
+message SocialLinks {
+  string youtube = 1;
+  string twitter = 2;
+  string twitch = 3;
+}
+
+message SiteBannerDetails {
+  string title = 1;
+  string description = 2;
+  string background_url = 3;
+  string logo_url = 4;
+  repeated int32 donation_amounts = 5;
+  SocialLinks social_links = 6;
+}
+
+enum WalletConnectedState {
```

Comment:
So right now it seems like we can only have one type of wallet connected. Is there plans to have a one account to many wallet support?

Reply:
I'll refactor this part completely